src/ResourceManager/Compute/Commands.Compute/packages.config

```diff
@@ -4,6 +4,10 @@
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Compute" version="16.3.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.KeyVault" version="2.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="16.0.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Storage" version="4.1.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
Please remove these common references - all are loaded in the common targets and thus don't need to be here.